### PR TITLE
add ado e2e, use ds for host exfiltration

### PR DESCRIPTION
**Pull request by alexeldeib**

### CODE REVIEW

File: e2e/e2e-script.sh [hunk 1]
```diff
@@ -51,61 +52,83 @@ MC_RESOURCE_GROUP_NAME=$(jq -r '.nodeResourceGroup' < cluster_info.json)
 VMSS_NAME=$(az vmss list -g $MC_RESOURCE_GROUP_NAME -ojson | jq -r '.[length -1].name')
 CLUSTER_ID=$(echo $VMSS_NAME | cut -d '-' -f3)
 
+# privileged ds with nsenter for host file exfiltration
+kubectl apply -f https://gist.githubusercontent.com/alexeldeib/01f2d3efc8fe17cca7625ecb7c1ec707/raw/6b90f4a12888ebb300bfb2f339cf2b43a66e35a2/deploy.yaml
```
Comment (alexeldeib) [inline e2e/e2e-script.sh:56]: could move to this repo instead of random gist

File: e2e/e2e-script.sh [hunk 2]
```diff
@@ -51,61 +52,83 @@ MC_RESOURCE_GROUP_NAME=$(jq -r '.nodeResourceGroup' < cluster_info.json)
...
-    addJsonToFile "$file" "$content"
-done
-tlsEndTime=$(date +%s)
-log "Retrieved TLS data in $((tlsEndTime-tlsStartTime)) seconds"
-
-# Add other relevant information needed by AgentBaker for bootstrapping later
+# az vmss run-command invoke \
+#             -n $VMSS_NAME \
+#             -g $MC_RESOURCE_GROUP_NAME \
+#             --command-id RunShellScript \
```
Comment (alexeldeib) [inline e2e/e2e-script.sh:83]: self nit: clean up in future

**Review by anujmaheshwari1 — APPROVED** [review]

File: .pipelines/e2e.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+name: $(Date:yyyyMMdd)$(Rev:.r)_e2e
+trigger:
+  branches:
+    include:
+    - ace/moe2e
```
Comment (anujmaheshwari1) [inline .pipelines/e2e.yaml:5]: Not sure I follow, why is there a specific branch in the trigger?

Comment (alexeldeib) [reply .pipelines/e2e.yaml:5]: it's the the branch I was developing on, whoops

**Review by anujmaheshwari1 — APPROVED** [review]

**Pull request merged**